geomesa-convert/geomesa-convert-scripting/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--********************************************************************
+* Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
```

Comment:
i don't think we have licenses in the poms - it's not checked by the plugin, at any rate